src/pages/dmt5/_margin_calculator.js

```diff
@@ -0,0 +1,275 @@
+import React from 'react'
+import Proptypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import { isMobile } from 'common/os-detect'
+import { isBrowser } from 'common/utility'
+import device from 'themes/device'
+import { Box, Flex, SectionContainer } from 'components/containers'
+import { Carousel, Header, LinkText, QueryImage, Text } from 'components/elements'
+import { LinkButton } from 'components/form'
+import { Localize } from 'components/localization'
+
+const query = graphql`
+    query {
+        margin_calculator: file(relativePath: { eq: "dmt5-margin-calculator.png" }) {
+            ...fadeIn
+        }
+        margin_calculator_mobile: file(relativePath: { eq: "dmt5-margin-calculator-mobile.png" }) {
+            ...fadeIn
+        }
+        swap_calculator: file(relativePath: { eq: "dmt5-swap-calculator.png" }) {
+            ...fadeIn
+        }
+        swap_calculator_mobile: file(relativePath: { eq: "dmt5-swap-calculator-mobile.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const ImageWrapper = styled.div`
+    max-width: 39.2rem;
+    width: 100%;
+    height: 38.6rem;
+    object-fit: contain;
+    margin-bottom: 2.4rem;
+
+    @media ${device.mobileL} {
+        max-width: 232px;
+        width: 100%;
+        margin: 0 0 24px;
+        height: unset;
+
+        div {
+            max-width: 232px;
+            width: 100%;
+        }
+    }
+`
+
+const MainHeader = styled(Header)`
+    margin: 0 0 12px;
+    @media ${device.tabletL} {
+        font-size: 32px;
+        margin-bottom: 24px;
+        padding: 0 16px;
+        text-align: center;
+    }
+`
+
+const SubHeader = styled(Header)`
+    font-size: 24px;
+    color: var(--color-black-3);
+    @media ${device.tabletL} {
+        font-size: 20px;
+        margin-bottom: 8px;
+    }
+`
+
+const StyledBox = styled(Box)`
+    max-width: 508px;
+    margin: 11.9rem 4rem 0 16rem;
+    @media ${device.tabletL} {
+        max-width: 100%;
+        margin: 40px 16px 24px;
+    }
+`
+
+const StyledText = styled(Text)`
+    text-align: ${(props) => (props.align ? 'center' : 'left')};
```

Comment:
we can change this props name to `is_centered` to avoid confusion because `align` should be used as `align=...` straight away